### ISSUE TITLE
Category parents completely ignored while importing using the plugin.

### ISSUE THREAD

**Issue by whoismark** [proteusthemes/one-click-demo-import#61]
Hey!

I noticed that while uploading a demo content with this plugin the category parents are completely ignored meaning every subcategory becomes a standalone category. This doesn't happen while importing the demo content with standard WordPress Importer, so it most likely is a plugin issue 😿  as I did everything according to docs.

I'd really appreciate you looking into it. Many thanks! Wonderful plugin 🏆

**Comment by capuderg** [comment]
Hi,

thanks for reporting this bug. I'll take a look at this next week, because I won't be able to work on it this weekend.

We are using the new WP importer 2.0: https://github.com/humanmade/WordPress-Importer, so probably the issue of parent category also exists in this new content importer...

Take care!

**Comment by whoismark** [comment]
The pleasure is all mine :)

Tested the WP importer 2.0 and the issue is also present there :( Moving to a previous *working* version would probably solve this issue. Let me know if there is any way I can help :)

**Comment by capuderg** [comment]
Thanks for testing the WP importer 2.0.

I think this parent category never worked in this new WP importer 2.0, so I'll have to take a look at how to fix this in our forked version of this importer...

**Comment by whoismark** [comment]
Thanks! Very much looking forward to it :) 👍

**Comment by capuderg** [comment]
Hi,

I've solved other bugs/features for this plugin, so I will be able to look at this by the end of this week. Just wanted to let you know (that I didn't forget about this) :)

Take care!

**Comment by whoismark** [comment]
Great, I was just thinking about this. Let me know if I can help ;)

**Comment by capuderg** [comment]
Hi!

I think I fixed the issue :)

I did some basic testing, but I would like you to test it as well.

Here is the OCDI release candidate 2.0.0: https://dl.dropboxusercontent.com/u/14641788/OCDI/one-click-demo-import-2.0.0-RC.zip

Please test it out and let me know, if it works ok with your theme import.

Take care!

**Comment by whoismark** [comment]
Yes! Works great now! Tested on my development site with many subcategories and, so far, everything seems to be working perfectly :) I'll let you know if I find some bug in the mean time. Any timeframe for the update? :)

**Comment by capuderg** [comment]
I hope I will be able to update the plugin tomorrow or on saturday. But it depends, if I find any last minute bugs...

**Comment by capuderg** [comment]
I'll close this issue for now. Please reopen it, if something comes up.

**Comment by capuderg** [comment]
Hi,

just a quick update. I found two bugs while testing, so the plugin will not be updated this weekend. :(

I've pushed back the update to next weekend... I hope I'll have time to work on it and fix the issues. The main problem is that the customizer import is not working ok on WP version 4.7-RC2. Since the WP 4.7 release is just around the corner I don't want to update the plugin, if it is not working 100% with WP 4.7.

Here is the latest RC for our OCDI plugin, if someone wants to do some tests: https://dl.dropboxusercontent.com/u/14641788/OCDI/OCDI-2.0.0-RC2.zip

Take care!
